### PR TITLE
[SPARK-29455][WEBUI] Improve tooltip information for Stages

### PR DESCRIPTION
**Pull request by sharangk**
### What changes were proposed in this pull request?
Adding tooltip to Stages tab for better usability.

### Why are the changes needed?
There are a few common points of confusion in the UI that could be clarified with tooltips. We
should add tooltips to explain.

### Does this PR introduce any user-facing change?
Yes
image

### How was this patch tested?
Manual

### CODE REVIEW

**Review by srowen — APPROVED** [review]
OK, I think that may be all there is to do here, yes.

File: core/src/main/scala/org/apache/spark/ui/jobs/StageTable.scala [hunk 1]
```diff
@@ -183,7 +183,8 @@ private[ui] class StagePagedTable(
       Seq(("Stage Id", null, true)) ++
       {if (isFairScheduler) {Seq(("Pool Name", null, true))} else Seq.empty} ++
       Seq(
-        ("Description", null, true), ("Submitted", null, true), ("Duration", null, true),
+        ("Description", null, true), ("Submitted", null, true),
```
Comment (seanli-rallyhealth) [inline core/src/main/scala/org/apache/spark/ui/jobs/StageTable.scala]: why not break to one tuple each line?

Comment (sharangk) [reply core/src/main/scala/org/apache/spark/ui/jobs/StageTable.scala]: Done

**Review by dongjoon-hyun — APPROVED** [review]
+1, LGTM. Merged to master.
Thank you, @sharangk , @srowen , @seanli-rallyhealth

**Comment by srowen** [comment]
The last test run actually failed here @dongjoon-hyun but I'm quite sure it's spurious. Let's watch the master build but I doubt there's any issue.

**Comment by dongjoon-hyun** [comment]
Oh. I missed. Thanks!

**Pull request closed without merging**